app/models/services/wordpress.rb

```diff
@@ -22,7 +22,7 @@ def post post, url=''
 
   def post_body post
     {
-     title: post.message.title(length: 40),
+     title: post.message.title,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.